packages/@webex/plugin-meetings/test/unit/spec/meeting/utils.js

```diff
@@ -378,24 +378,38 @@ describe('plugin-meetings', () => {
     });
 
     describe('joinMeeting', () => {
-      it('#Should call `meetingRequest.joinMeeting', async () => {
-        const meeting = {
+      const joinMeetingResponse = {
+        body: {
+          mediaConnections: [],
+          locus: {
+            url: 'differentLocusUrl',
+            self: {
+              id: 'selfId',
+            },
+          },
+        },
+        headers: {
+          trackingid: 'trackingId',
+        },
+      }
+      let meeting
+
+      beforeEach(() => {
+        meeting = {
           meetingJoinUrl: 'meetingJoinUrl',
           locusUrl: 'locusUrl',
           meetingRequest: {
             joinMeeting: sinon.stub().returns(
-              Promise.resolve({
-                body: {mediaConnections: 'mediaConnections'},
-                headers: {
-                  trackingid: 'trackingId',
-                },
-              })
-            ),
+              Promise.resolve(joinMeetingResponse))
           },
           getWebexObject: sinon.stub().returns(webex),
-        };
+          locusInfo: {
+            initialSetup: sinon.stub(),
+          },
+        }
+      })
 
-        const parseLocusJoinSpy = sinon.stub(MeetingUtil, 'parseLocusJoin');
+      it('#Should call `meetingRequest.joinMeeting', async () => {
```

Comment:
Only this test was changed so locusInfo is tested.
Because locusInfo is now required for joinMeeting to work with tests, `meeting` object was moved one level (to `beforeEach`) to remove unnecessary duplication

Reply:
Just got a quick question about the tests here, why are we having a `#` here?

Reply:
looks like the other existing tests nearby also already have `#` at the start of the title, they're all unnecessary. From what I've seen, the SDK tests usually have a convention to add a `#` at the beginning of method name when creating a `describe` for tests for such method, so in this case we should have `describe('#joinMeeting'), () => {` in line 380 and not have any `#` in the `it()` calls